src/Views/Form/Templates/Classic/resources/css/form.scss

```diff
@@ -29,3 +29,6 @@
 @import 'donation-amount';
 @import 'personal-info';
 @import 'payment-details';
+
+// Receipt
+@import "receipt";
```

Comment:
I’m thinking that it might be good to make this a completely separate stylesheet that shares styles with the form css. This works for now though.

Reply:
Yeah, good idea. We will revisit all this moving forward to Give 3.0.